tst/EnergyPlus/unit/SurfaceGeometry.unit.cc

```diff
@@ -8661,10 +8661,6 @@ TEST_F(EnergyPlusFixture, GetSurfaceData_SurfaceOrder)
 
     SetupZoneGeometry(*state, ErrorsFound);
     EXPECT_FALSE(ErrorsFound); // expect no errors
-
-    GetSurfaceData(*state, ErrorsFound); // setup zone geometry and get zone data
```

Comment:
This function is already called from `SetupZoneGeometry`. Otherwise, no substantive changes to the exisiting `GetSurfaceData_SurfaceOrder` unit test.